CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES ".*Clang")
-    set(CMAKE_CXX_FLAGS "-Wno-unused-parameter -Werror ${CMAKE_CXX_FLAGS}")
+    set(CMAKE_CXX_FLAGS "-Wno-unused-parameter -Wno-unused-command-line-argument -Werror ${CMAKE_CXX_FLAGS}")
```

Comment:
What is triggering the unused-command-line-argument warning?  If it's something we add in cmake, it might be worth fixing that instead of ignoring it...

Reply:
I think it's fine to remove `-msse2`, as that will be on by default for all amd64 targets (and I don't think we need to worry about 32-bit x86 on Linux or macOS any more).  It also won't apply to other architectures (e.g. Arm64) anyway

Reply:
Removed! 👍🏼